tests/test_course_meta_loader.py

```diff
@@ -0,0 +1,125 @@
+from raise_data.loaders import course_meta_loader
+import boto3
+import pytest
+import botocore.stub
+import csv
+from io import StringIO, BytesIO
+from sqlalchemy.dialects.postgresql import insert
+from raise_data.dashboard.schema import Course
+
+
+@pytest.fixture(autouse=True)
+def setup_database_table():
+    """This fixture clears and populates the Course table before each test
+    run in this file
+    """
+    with course_meta_loader.session_factory.begin() as session:
+        session.query(Course).delete()
+        insert_stmt = insert(Course).values(
+            [
+                {'id': 1, 'name': 'Name 1', 'term': 'term1', 'district': ''},
+                {'id': 2, 'name': 'Name 2', 'term': 'term1', 'district': None}
+            ]
+        )
+        session.execute(insert_stmt)
+
+
+def test_course_meta_loader(mocker):
+
+    s3_client = boto3.client("s3")
+    s3_stubber = botocore.stub.Stubber(s3_client)
+
+    mock_dict_writer_data_1 = [
+        {
+            "course_id": 1,
+            "district": "Halo School District",
+        },
+        {
+            "course_id": 2,
+            "district": "Azeroth School District",
+        },
+        {
+            "course_id": 3,
+            "district": "Faerun School District"
+        }
+    ]
+
+    mock_course_metadata_bytes = StringIO()
+    mock_fieldnames = mock_dict_writer_data_1[0].keys()
+    writer = csv.DictWriter(
+        mock_course_metadata_bytes, fieldnames=mock_fieldnames
+        )
+    writer.writeheader()
+    writer.writerow(mock_dict_writer_data_1[0])
+    writer.writerow(mock_dict_writer_data_1[1])
```

Comment:
Super minor: You can do this in one line as `writer.writerows(mock_dict_writer_data_1)`